Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.cpp

```diff
@@ -139,7 +139,8 @@ void RemoteGraphicsContextGL::workQueueInitialize(WebCore::GraphicsContextGLAttr
 void RemoteGraphicsContextGL::workQueueUninitialize()
 {
     assertIsCurrent(workQueue());
-    m_context->setClient(nullptr);
+    if (m_context)
+        m_context->setClient(nullptr);
     m_context = nullptr;
```

Comment:
This is the only intended place that currently clears the m_context, is there's something strange going on if it can already be nullptr. Could you investigate that still?

Reply:
This was left over from another change. Removed.